vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -74,10 +78,20 @@ var EsxPort int
 func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) ([]byte, error) {
 	vmdkCmd.Mtx.Lock()
 	defer vmdkCmd.Mtx.Unlock()
-
+	var versionEnv = os.Getenv("VMCI_VERSION")
+	log.Debugf("Run get request: version=%s", versionEnv)
+	// TODO: versionEnv need to read from enviroment variable when Nirdesh's Makefile is ready
```

Comment:
please do not leave names in comments. It's better to open an issue and simply refer it.